qtopconf.yaml

```diff
@@ -21,10 +21,15 @@ schedulers:
     oarnodes_y_file: %(savepath)s/oarnodes_Y%(pid)s.txt, oarnodes -Y
     oarstat_file: %(savepath)s/oarstat%(pid)s.txt, oarstat
   sge:
-    sge_file_stat: %(savepath)s/qstat%(pid)s.F.xml.stdout, qstat -F -xml -u '*'
+    sge_file: %(savepath)s/qstat%(pid)s.F.xml.stdout, qstat -F -xml -u '*'
   demo:
     demo_file: %(savepath)s/demo%(pid)s.txt, echo 'Demo here'
 
+signature_commands:
+  pbs: pbsnodes
+  oar: oarnodes
+  sge: qhost
```

Comment:
after reading this: http://cc.in2p3.fr/docenligne/970 , I know believe again qacct is better ...
(eh, I know, I change my mind all the time!)